src/group/mls_group/test_mls_group.rs

```diff
@@ -194,13 +173,6 @@ fn test_update_path() {
                 false,
             )
             .expect("Error creating commit");
```

Comment:
Why did you remove the checks here? Not super important but they are still nice to have.

Reply:
The exact same procedure was copied from another test, which does those checks. I thought I'd simplify the group setup, but I'm happy to add them back.